stock_picking_jobcan_wf_confirm/models/stock_picking_jobcan_wf_confirm.py

```diff
@@ -0,0 +1,100 @@
+import logging
+
+from odoo import _, api, models
+from odoo.exceptions import UserError
+
+_logger = logging.getLogger(__name__)
+
+
+class StockPickingJobcanWfConfirm(models.Model):
+    _name = "stock.picking.jobcan.wf.confirm"
+    _description = "Stock Picking Jobcan Workflow Confirm"
+    _inherit = ["api.call.mixin"]
```

Comment:
I suppose we don't need to define a new model.  It's probably more appropriate to just extend `stock.picking`.